spec/spec_helper.rb

```diff
@@ -1,19 +1,12 @@
 ENV['RACK_ENV'] = 'test'
-
-require 'bundler'
-
-Bundler.setup
-Bundler.require(:default, ENV['RACK_ENV'])
-
-require 'find'
-%w(./config/initializers ./lib).each do |load_path|
-  Find.find(load_path) { |f| require f if f.match(/\.rb$/) }
-end
+require_relative '../config/boot'
 
 # Load support files
 Dir[File.dirname(__FILE__) + '/support/*.rb'].each { |f| require f }
 
 RSpec.configure do |config|
+  config.include JsonSpec::Helpers
```

Comment:
In the recurrence service I have a spec group (i.e. `type: :request`) to include stuff like this only in request specs.